app/views/user_mailer/form_feedback.text.erb

```diff
@@ -0,0 +1,8 @@
+You recently archived the form, <%= @form.title %> in Touchpoints.
+
+The form was created <%= time_ago_in_words(@form.created_at) %> ago.
+The form received <%= @form.response_count %> responses.
+
+Please take this <%= link_to "short survey", @feedback_url %> to provide feedback regarding the Touchpoints application and/or the Touchpoints team.
```

Comment:
Is it supposed to work if you put an anchor tag in the `text/plain` version?

In Mailcatcher, at least, it looks weird.
<img width="1362" alt="Screenshot 2024-06-11 at 2 39 45 PM" src="https://github.com/GSA/touchpoints/assets/8216592/9d288341-ef00-4879-af40-b4317a9fc52d">

Reply:
this has been addressed, only showing 'short survey' as text and relying on the URL below.